asset/asset.go

```diff
@@ -427,6 +431,23 @@ type GroupKey struct {
 	Sig schnorr.Signature
 }
 
+// GroupKeyReveal is a type for representing the data used to derive the tweaked
+// key used to identify an asset group. The final tweaked key is the result of:
+// TapTweak(internalTweakedKey, tapscriptRoot)
```

Comment:
nit: inconsistent with naming below

Reply:
Aligned names to those used in the godoc for the `GroupKey` struct.